Slim/Menu/BrowseLibrary/Releases.pm

```diff
@@ -131,15 +154,15 @@ sub _releases {
 		"library_id:$library_id",
 	];
 
-	if (delete $contributions{COMPOSER}) {
+	if (my $albums = delete $contributions{COMPOSER}) {
```

Comment:
Another duplicate variable name... and I did that for the `TRACKARTIST`, too. Please rename both to something like `$albumIDs` instead.

Reply:
Yeah, I copied your code from TRACKARTIST when I needed the album list in COMPOSERS!